env/dev/utils.clj

```diff
@@ -0,0 +1,8 @@
+(ns utils)
+
+(defn deep-merge
```

Comment:
https://github.com/status-im/status-react/blob/develop/src/status_im/utils/utils.cljs#L137

Reply:
Yes it is the same `deep-merge` function from `status-im.utils.utils`, I simply made a copy of it so that it can be used by `clj`. Maybe it is better to move to a shared namespace which both `clj` and `cljs` can access?

Reply:
Yes, please move it (and other candidates) to `status-im.utils.core.cljc` :)